geonode/base/api/tests.py

```diff
@@ -283,6 +283,46 @@ def test_base_resources(self):
             response = self.client.get(f"{url}/{layer.id}/", format='json')
             self.assertTrue('links' in response.data['resource'].keys())
 
+        # test 'tkeywords'
+        try:
+            for _tkw in ThesaurusKeyword.objects.all()[:2]:
+                resource.tkeywords.add(_tkw)
+            self.assertEqual(2, resource.tkeywords.count())
+            # Admin
+            self.assertTrue(self.client.login(username='admin', password='admin'))
+            response = self.client.get(f"{url}/{resource.id}/", format='json')
+            self.assertIsNotNone(response.data['resource']['tkeywords'])
+            self.assertEqual(2, len(response.data['resource']['tkeywords']))
+            self.assertListEqual(
+                [
+                    {
+                        'name': '',
+                        'slug': 'http-inspire-ec-europa-eu-theme-37',
+                        'uri': 'http://inspire.ec.europa.eu/theme#37',
+                        'thesaurus': {
+                            'name': 'GEMET - INSPIRE themes, version 1.0',
+                            'slug': 'inspire-theme',
+                            'uri': 'http://inspire.ec.europa.eu/theme'
+                        },
+                        'i18n': {}},
```

Comment:
To have a more complete test, here should be useful to have also the translations and the `name` field filled

Reply:
Done, thanks